tests/plugins/contextmenu/manual/enabledNativeSpellChecker.md

```diff
@@ -0,0 +1,12 @@
+@bender-tags: 4.5.0, enhancement
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, elementspath, resize
+
+This test checks whether, when a word is misspelled, that when right clicking, suggestions will appear.
+
+1. Misspell word.
+2. Right click.
+3. Locate correctly spelled word.
+4. Choose correctly spelled word.
+
+**Expected result:** After choosing correctly spelled word, spell check suggestion disappears.
```

Comment:
This basically checks native browser behaviour (spellchecker suggestions in native browser menu) which we don't have full control over. It will be good to redefine this test so it checks CKEditor 4 behaviour only (e.g. simply if native context menu is shown).